src/sage/features/symengine_py.py

```diff
@@ -0,0 +1,43 @@
+r"""
+Check for symengine_py
+"""
+
+# ****************************************************************************
+#       Copyright (C) 2023 Dima Pasechnik
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#                  https://www.gnu.org/licenses/
+# ****************************************************************************
+
+from . import PythonModule
+from .join_feature import JoinFeature
+
+
+class symengine_py(JoinFeature):
```

Comment:
I think this should be just a `PythonModule`-feature (instead of `JoinFeature`).

Reply:
No, it has to be JoinFeature because the feature cannot be "symengine" (that is already taken by the C package)

Reply:
It's not possible to decouple the feature name from the python module name?

Reply:
It's one of the use cases of JoinFeature.